resources/syntax/Twig.sublime-syntax

```diff
@@ -103,6 +105,10 @@ contexts:
     - match: \b(if)\b
       scope: keyword.control.conditional.if.twig
 
+  match_keyword_endif:
+    - match: \b(endif)\b
+      scope: keyword.control.conditional.endif.twig
```

Comment:
IMO, this should be scoped as `keyword.control.conditional.end.twig`. Please have a look at this comment :- https://github.com/sublimehq/Packages/issues/1228#issuecomment-461077685